### DISCUSSION TITLE
使用 SiteInfo 的页面会出现错误

### DISCUSSION THREAD

**Discussion by Masterain98** [vuepress-theme-hope/vuepress-theme-hope#2765] [Q&A] [answered]
参照着文档站的[这个页面](https://theme-hope.vuejs.press/zh/demo.html)做了两个页面
- https://uigf.org/  （使用`@include`包含了partnership-index.html）
- https://uigf.org/partnership.html
- 代码在：https://github.com/UIGF-org/UIGF-org.github.io/tree/main/docs

## 问题
1. 会出现重复的站点
![image](https://user-images.githubusercontent.com/10614984/218255791-7cb1d9b5-865c-44a1-9fda-779a7a822c74.png)

2. 网页边缘会出现“溢出”的 GitHub 按钮
![image](https://user-images.githubusercontent.com/10614984/218255819-452eb524-dd48-4305-8b1a-9e77353a7c0a.png)

如果是通过网页内链点击到相关页面，就不会出现问题。但如果是输入 URL 或者重新刷新页面，问题就会出现

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/9ff243fdd029b56505563047edbc65f22d0ba82f
